src/styles/main.scss

```diff
@@ -0,0 +1,83 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
```

Comment:
do not use `*`  selector to reset paddings or margins

use some specific tags for it